webknossos/webknossos/dataset/dataset.py

```diff
@@ -2202,6 +2206,18 @@ def allowed_teams(self, allowed_teams: Sequence[Union[str, "Team"]]) -> None:
                 self._organization_id, self._dataset_name, team_ids
             )
 
+    def explore_and_add_remote(
```

Comment:
I think this should return the RemoteDataset that has been created.

Reply:
Okay, this should work now.